### DISCUSSION TITLE
Fund Stolen

### DISCUSSION THREAD

**Discussion by mazher069** [blockscout/blockscout#9097] [General]
Respected Team
i have request to check my address on ETHERSCAN AIRBITSCAN and also PACIFIC EXPLORER MANTA yesterday same time around 4PM PST all  my funds from the address ( 0x7837a8660c595382015D806201013C32cB8B9E60 ) were stolen 
in my METAMASK wallet permission were approved to 
LAYERBANK
SYMBIOS
and all other swap platform which  mention on PARADIGM site
around 0.03452344 ETH from ETHERIOUM main chain
Transaction Hash
0xa9c475b3c735e1d6dc24c6b21c9e9044099b90c591dd3953ee9f46bb1b9e6c0b

around 0.00605655492 ETH from AIRBITRUM main chain
Transaction Hash
0x2901adf722ffa9bf0baaddd39b599098b2c7ab5da939824c924b31b5726eb6e4

around 0.65 stone + 15$ manta eth from PACIFIC EXPLORER OF MANTA
Transaction Hash
0x6a1e37e6f4d10d4db80f38eb1e1896a7e730f07b1732a51f8f4baf7f4983fc0f

 and the common thing is the address all this send to the following address (AIRBITRUM and ETHERIOUM chain )
0xf29DDfF7E3aE49FD98c575754557bbe4f766061E
0xf29ddff7e3ae49fd98c575754557bbe4f766061e

on PACIFIC EXPLORER OF MANTA the address is and in the start CORE is mention plz guide me about this issue i am very disheart after all this plz team help me regards this issue...
0xB7A23Fc0b066051dE58B922dC1a08f33DF748bbf

**Comment by Mojmir-R** [comment]
Super sorry to hear this but unfortunately there is absolutely nothing that we, as a Blockscout team, can do to help you with this.

**Reply by mazher069** [reply]
PLZ CHECK AND UPDATE ME TOO
JUST BECOUSE SHARE ALL HASHES TOO

**Reply by CharleneeveRivera** [reply]
Sorry about this buddy, 
You can send the transaction hash and address to cryptorecoverycenter@outlook.com 

An agent will be assigned to your case and you will be provided a solution.

**Reply by Martincho-c** [reply]
Yo tengo una recepcion de criptos pendiente hace 5 días y nadie me da una repuesta desde donde me enviaron las criptos no se reintegro y en mi walet está pendiente ...

**Comment by Mohammadkhan0** [comment]
![Screenshot_20240508_162119_com haqq wallet](https://github.com/blockscout/blockscout/assets/147714864/32cb91e0-0d7b-41e0-91fa-520aaa7ae2c9)

**Reply by Mohammadkhan0** [reply]
I wrongly transfer my 138 islm coin to other address as per proof shown in attached, I want my 138 islm coin back on my source address on
0xca7683d256ccd802fbbf54ecde0ac88a2a4cfe98
